CHANGELOG.asciidoc

```diff
@@ -36,6 +36,7 @@ Notes:
 [float]
 ===== Features
 
+* Adds support for Amazon SQS queues via `aws-sdk` instrumentation.
```

Comment:
A link to the message-queues doc here would be very helpful, especially given there is some nuance to SQS instrumentation.

I haven't looked at the queue delay metrics yet, but I'm guessing it would be good to mention their being added in the changelog as well.

Reply:
_nod_ that makes sense.  I've added a link to messaging spec and called out the queue metrics by name.